src/librustc_trans/back/lto.rs

```diff
@@ -82,16 +84,11 @@ pub fn run(sess: &session::Session,
     // For each of our upstream dependencies, find the corresponding rlib and
     // load the bitcode from the archive. Then merge it into the current LLVM
     // module that we've got.
-    link::each_linked_rlib(sess, &mut |cnum, path| {
-        // `#![no_builtins]` crates don't participate in LTO.
-        if sess.cstore.is_no_builtins(cnum) {
```

Comment:
Yeah this query just ended up having a lot of dependencies on `sess` so I figured it'd be best to move it way up to the beginning instead of only running it back here.